source/content/drupal-10.md

```diff
@@ -12,15 +12,29 @@ product: [composer]
 integration: [--]
 ---
 
-Drupal 10 is coming in December 2022! We've been busy conducting early testing on the Pantheon platform, and have developed a process for creating a new Drupal 10 site on our platform.
+Drupal 10 is coming in December 2022! This page will be updated frequently with the latest information and instructions for Drupal 10 on Pantheon.,
+
+## Current Status of Drupal 10 at Pantheon
+
+| Scenario | Status | Documentation |
+|---|---|---|
+| Create a Drupal 10 site on Pantheon | Available | [Create a Drupal 10 Site](/drupal-10#create-a-drupal-10-site)|
+| Upgrade a Drupal 9 site to Drupal 10 manually | Available | [Upgrade from Drupal 9 to Drupal 10](/drupal-10#upgrade-a-drupal-9-site-to-drupal-10) |
+| Create Drupal 10 site from Pantheon Dashboard | Planned availability January 2023 | |
+| Upgrade a Drupal 9 site to Drupal 10 using the [Terminus Conversion Tools plugin](https://github.com/pantheon-systems/terminus-conversion-tools-plugin) | Early Access | Use the following command: `terminus conversion:upgrade-d10` |
+
+
+## Create a Drupal 10 Site
+
+We've been busy conducting early testing on the Pantheon platform, and have developed a process for creating a new Drupal 10 site on our platform.
 
 <Alert title="Warning" type="danger" >
 
 These instructions were developed using beta versions of Drupal 10, and are for educational purposes only.  This page will be updated as production versions become available.
 
 </Alert>
 
-## Before You Begin
+### Before You Begin
 
 This process requires that you are on PHP 8.1.  To upgrade:
```

Comment:
Drupal (Composer Managed) will be on PHP 8.1 later today. Let's change this to provide instructions for folks with existing sites, and tell them they should be good to go if they create a new site.

If the user sees an update offered on the dashboard to go to PHP 8.1, they should apply that, and it will do the right thing for them.

If they don't see that update, then maybe they are on a different upstream. They can follow steps 2 - 4 below (rewrite without step 1, so existing step 2 becomes new step 1).  They will also need to modify their [platform php configuration](https://github.com/pantheon-upstreams/drupal-composer-managed/blob/main/composer.json#L84) in their top-level composer.json to read "8.1.13" instead of "7.4". Add this as a new step at the end.